jha/src/views/enrolment/SpouseInfoPage.vue

```diff
@@ -15,9 +16,15 @@
         <div class="text-danger"
           v-if="$v.hasSpouse.$dirty && !$v.hasSpouse.required"
           aria-live="assertive">Please indicate whether or not you have a spouse who needs to enrol.</div>
+        
         <div v-if="hasSpouse === 'Y'" class="mt-3">
           <h2>Spouse or common-law partner</h2>
-          <p>Please provide the spouse's personal information and immigration status in Canada. You will be required to upload supporting documents with your application.</p>
+          <div class="heading mt-3">
+            <p>Please provide the spouse's personal information and immigration status in Canada. You will be required to upload supporting documents with your application.</p>
+            <div class="ml-1 remove-icon" @click="removeSpouse()">
+              <font-awesome-icon icon="times-circle" size="2x"/>
+            </div>
```

Comment:
You might need to make the "remove-icon" a link in order for it to be keyboard focusable.